x/slashing/client/rest/query.go

```diff
@@ -75,3 +118,47 @@ func queryParamsHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext) http.Hand
 		rest.PostProcessResponse(w, cdc, res, cliCtx.Indent)
 	}
 }
+
+func getSigningInfo(cliCtx context.CLIContext, storeName string, cdc *codec.Codec, address []byte) (signingInfo slashing.ValidatorSigningInfo, code int, err error) {
+	key := slashing.GetValidatorSigningInfoKey(sdk.ConsAddress(address))
+
+	res, err := cliCtx.QueryStore(key, storeName)
+	if err != nil {
+		code = http.StatusInternalServerError
+		return
+	}
+
+	if len(res) == 0 {
+		code = http.StatusNoContent
+		return
+	}
+
+	err = cdc.UnmarshalBinaryLengthPrefixed(res, &signingInfo)
+	if err != nil {
+		code = http.StatusInternalServerError
+		return
+	}
+
+	return
+}
+
+func adjustPagination(size int, page int, limit int) (int, int) {
```

Comment:
All these arguments and return values should be `uint`

Reply:
I can change it here and cast when calling, I can't change it easily in ParseHTTPArgs as they are then forwarded to `node.TxSearch(query, prove, page, limit)` that's asking for `int`